clean.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+
+usage()
+{
+    echo "Usage: clean [options]"
+    echo "Cleans the local dev environment."
+    echo
+    echo "  -b     Delete bin directory"
+    echo "  -p     Delete packages directory and NuGet package caches"
+    echo "  -a     Remove all untracked files in the working tree and delete NuGet package caches"
```

Comment:
Agreed I like -all for a combinations of all and something else perhaps -s to clean the source working tree.

Personally I have a tendency of cleaning the bin dire and working tree but rarely ever clean the package cache.